README.md

```diff
@@ -4,7 +4,8 @@
 </p>
 
 <p align="center">
-  <a href="https://github.com/actions/toolkit"><img alt="GitHub Actions status" src="https://github.com/actions/toolkit/workflows/toolkit-unit-tests/badge.svg"></a>
+  <a href="https://github.com/actions/toolkit/actions?query=workflow%3Atoolkit-unit-tests"><img alt="GitHub Actions status" src="https://github.com/actions/toolkit/workflows/toolkit-unit-tests/badge.svg"></a>
```

Comment:
The diff is hard to see, i appended `/toolkit_/actions?query=workflow%3Atoolkit-unit-tests` to the existing link